src/coreclr/jit/fgprofilesynthesis.h

```diff
@@ -45,7 +45,7 @@ class ProfileSynthesis
     {
     }
 
-    static constexpr weight_t exceptionScale        = 0.001;
+    static constexpr weight_t exceptionWeight       = 0.00001;
```

Comment:
I'm a bit surprised to see this fixed weight is smaller than the scale factor it's replacing. Did you get this number through trial and error, or?

Reply:
Nothing magic here. Just wanted a value that is small enough to ignore in an absolute sense.

Reply:
I see -- so close to zero, but it won't mark the block with `BBF_RUN_RARELY`.